test/e2e/app-dir/dynamic-in-generate-params/index.test.ts

```diff
@@ -6,30 +6,20 @@ async function getLastModifiedTime(next: NextInstance, pathname: string) {
 }
 
 describe('app-dir - dynamic in generate params', () => {
-  const { next, isNextDev } = nextTestSetup({
+  const { next } = nextTestSetup({
     files: __dirname,
   })
 
   it('should render sitemap with generateSitemaps in force-dynamic config dynamically', async () => {
-    const firstTime = await getLastModifiedTime(
-      next,
-      isNextDev ? 'sitemap.xml/0' : '/sitemap/0.xml'
-    )
-    const secondTime = await getLastModifiedTime(
-      next,
-      isNextDev ? 'sitemap.xml/0' : '/sitemap/0.xml'
-    )
+    const firstTime = await getLastModifiedTime(next, 'sitemap/0')
+    const secondTime = await getLastModifiedTime(next, 'sitemap/0')
 
     expect(firstTime).not.toEqual(secondTime)
   })
 
   it('should be able to call while generating multiple dynamic sitemaps', async () => {
-    expect(
-      (await next.fetch(isNextDev ? 'sitemap.xml/0' : '/sitemap/0.xml')).status
-    ).toBe(200)
-    expect(
-      (await next.fetch(isNextDev ? 'sitemap.xml/1' : '/sitemap/1.xml')).status
-    ).toBe(200)
+    expect((await next.fetch('sitemap/0')).status).toBe(200)
+    expect((await next.fetch('sitemap/1')).status).toBe(200)
```

Comment:
Should we add assertions now to all of these pages that used to end with `.xml` that we send the correct content-type header?

Reply:
I added content-type assertions here. We also have it in metadata-dynamic-routes tests, it's double insurance now